pallets/loans/src/benchmarking.rs

```diff
@@ -457,6 +462,7 @@ benchmarks! {
 			let loan_id = (idx + 1).into();
 			let (loan_owner, asset) = create_asset::<T>(loan_id);
 			LoansPallet::<T>::create(RawOrigin::Signed(loan_owner.clone()).into(), pool_id, asset).expect("loan issue should not fail");
+			// (Remove this comment) Maybe an issue here.
 			activate_test_loan_with_defaults::<T>(pool_id, loan_id, loan_owner);
```

Comment:
Seems like there is an issue here. We suppose that the just created loan has the `loan_id` we have created above. It seems not to be always true because sometimes the loan is already priced at this point (which shouldn't), and I get an error because the `loan_owner` has not the `LoanAdmin` role for pricing on an active loan. I have "fixed" it by adding that role, but I think we should know why sometimes the loan is already priced at this point.

A noob question, is the pallet storage reset between benchmarks of the same `benchmarks!` section? That could be the problem because the same code is used in different benchmarks. The first benchmark will match the `loan_id` with the created loan, but the second does not and will reprice the first active loans.

Reply:
Is this still relevant, or was this fixed already?

Reply:
@branan or @mustermeiszer I do not know if any of you can take a look at this before merging the PR 🤗